index.html

```diff
@@ -4,7 +4,64 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
     <title>setup-mb-skeleton</title>
+    <link rel="stylesheet" href="style.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1>My logo</h1>
+      <img src="images/Menu.svg" alt="hamburger" />
+    </header>
+    <main class="wrapper">
+      <div class="banner">
+        <img src="images/Ellipse5.svg" alt="" />
+        <img src="images/Ellipse15.svg" alt="" class="top-circ icone-btn" />
+        <img src="images/Rectangle7.svg" alt="" />
+        <img src="images/Ellipse11.svg" alt="" class="aside-elip icone-btn" />
+      </div>
+      <section class="container">
+        <p class="primary-text">I'm Christian<br />Glad to see<br />you!</p>
+        <img src="images/Ellipse3.svg" alt="" class="big-elips icone-btn" />
+        <img src="images/Rectangle4.png" alt="" class="big-rect icone-btn" />
+        <p class="details">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <img src="images/Ellipse12.svg" alt="" class="btm-elips icone-btn" />
+        <img src="images/Ellipse13.svg" alt="" class="btm-mic icone-btn" />
+        <span class="connect">let's connect</span>
+        <div class="social">
+          <a href="#"><img src="images/twitter).svg" alt="twitter-icon" /></a>
```

Comment:
- [ ]Please make sure the margins  (top, bottom, left, right) are the same as defined in the design

 Your design  | Figma design
------------- | -------------
<img width="795" alt="Screenshot 2022-03-17 at 15 55 34" src="https://user-images.githubusercontent.com/77038610/158803443-d4dfad88-5f6a-4224-b260-44ac64595c14.png"> | <img width="325" alt="Screenshot 2022-03-17 at 15 56 12" src="https://user-images.githubusercontent.com/77038610/158803562-8a4a76c7-1b2a-44d6-86f1-3915b4ef0267.png">